include/grpc/support/log.h

```diff
@@ -28,31 +28,39 @@
 extern "C" {
 #endif
 
-/**
- * Logging functions in this file are deprecated.
- * Please use absl ABSL_LOG instead.
- */
+#define GPR_LOCATION __FILE__, __LINE__
+
+/** Should only be used from gRPC PHP and RUBY. **/
```

Comment:
this requires a gRFC

Reply:
I will try to find some other folder/file to put the new functions/macros - other than includes. Any pointers would help.
If that does not work, I will add some changes to the RFC.

Reply:
i'm unsure but check with alex/stanley if a src folder works

Reply:
Alex said that it won't work any other way unless we put in non trivial amount of effort. I will prepare the RFC and upload it within 90 minutes.
I guess we have to wait a week or two before submitting the RFC and then this PR. 
If you have other concerns in the PR content please let me know - so I can fix them.
@yashykt

Reply:
RFC https://github.com/grpc/proposal/pull/450